File: CS-Examples/01_MIMIE Message/ChangeEmailAddress.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using Spire.Email;

namespace ChangeEmailAddress
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void btnRun_Click(object sender, EventArgs e)
        {
            MailMessage mail = MailMessage.Load(@"..\..\..\..\..\..\Data\Sample.eml");
            // A To address with a friendly name can also be specified
            mail.To.Add(new MailAddress("To@domain.com", "e-iceblue"));
            // Specify Cc email address along with a friendly name
            mail.Cc.Add(new MailAddress("Cc@domain.com", "ExpectedName"));
            mail.Save("ChangeEmailAddress.eml", MailMessageFormat.Eml);
            MessageBox.Show("Completed");
        }
    }
}

File: VB-Examples/01_MIMIE Message/ChangeEmailAddress.vb
Imports System.ComponentModel
Imports System.Text
Imports Spire.Email

Namespace ChangeEmailAddress
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub btnRun_Click(ByVal sender As Object, ByVal e As EventArgs) Handles btnRun.Click
			Dim mail As MailMessage = MailMessage.Load("..\..\..\..\..\..\Data\Sample.eml")
			' A To address with a friendly name can also be specified
			mail.To.Add(New MailAddress("To@domain.com", "e-iceblue"))
			' Specify Cc email address along with a friendly name
			mail.Cc.Add(New MailAddress("Cc@domain.com", "ExpectedName"))
			mail.Save("ChangeEmailAddress.eml", MailMessageFormat.Eml)
            MessageBox.Show("Completed")
		End Sub
	End Class
End Namespace